GitHubDeployments.ps1

```diff
@@ -0,0 +1,567 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+@{
+    GitHubDeploymentEnvironmentTypeName = 'GitHub.DeploymentEnvironment'
+}.GetEnumerator() | ForEach-Object {
+    Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
+}
+
+filter New-GitHubDeploymentEnvironment
+{
+    <#
+    .SYNOPSIS
+        Creates a new deployment environment on a GitHub repository.
```

Comment:
Given that you are aliasing this as both `New` _and_ `Set`, you may need to update the documentation language to match (e.g. "Creates a new (or updates an existing) ...") (or use the API docs method of "Creates or updates an ...").  Here and elsewhere.